pallas-crypto/src/vrf/mod.rs

```diff
@@ -0,0 +1,159 @@
+use crate::hash::Hash;
+use crate::memsec::Scrubbed;
+use thiserror::Error;
+use vrf_dalek::vrf03::{PublicKey03, SecretKey03, VrfProof03};
+
+#[derive(Error, Debug)]
+pub enum Error {
+    #[error("VrfError {0}")]
+    VrfError(#[from] vrf_dalek::errors::VrfError),
+}
```

Comment:
This is me being picky again, the error type could be simplified and we should take ownership of the error handling:

This error is only for the verification (and I like that we have one error for the VRF Verification and not one for both the PublicKey decoding, PrivateKey decoding etc...).

How about we make the error a bit more explicit to clarifify the API:

```suggestion
/// error that can be returned if the verification of a [`VrfProof`] fails
/// see [`VrfProof::verify`]
///
#[derive(Error, Debug)]
#[error("VRF Proof Verification failed.")]
pub struct VerificationError(
    #[from] // so we keep the `?` handling in the function as you used
    #[source] // it is included with the `#[from]` so you can ignore tbh
    vrf_dalek::errors::VrfError
);
```